check.sh

```diff
@@ -78,10 +78,10 @@ fi
 
 # Check pip compile is consistent
 echo "::group::Pip Compile - Tests"
-uv pip compile --no-strip-markers --python-version=3.8 test-requirements.in -o test-requirements.txt
+uv pip compile --universal --no-strip-markers --python-version=3.8 test-requirements.in -o test-requirements.txt
```

Comment:
I think `--universal` implies `--no-strip-markers` but I haven't checked yet. (I was checking which is why I noticed the duplicated requirements entries before)

Reply:
Can't seem to find `--universal` mentioned anywhere in the docs, so I'm not sure either.

Reply:
Yeah the docs are out of date. I suspect they'll stay this way at least until uv actually releases their "uv tool" thing.

Reply:
Removed `--no-strip-markers` in d3b388c